shared_libs/hl7_validation/tests/test_paris.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+
+from hl7_validation import validate_er7_with_flow
+from hl7_validation.validate import XmlValidationError
+
+
+class TestParisValidation(unittest.TestCase):
+    def test_paris_a05_schema_validation_working(self) -> None:
```

Comment:
[nit] schema validation should always be working - it can be successful or fail, would rename these two scenarios to validation_successful

Reply:
also in the naming, here you use the A05 message structure but in the failure scenarios it's the trigger event A28 - which one's better for consistency?